azure/posix.yml

```diff
@@ -64,7 +61,7 @@ jobs:
       - bash: |
           set -e
           pip install virtualenv
-          BUILD_DEPENDS="$NP_BUILD_DEP $CYTHON_BUILD_DEP $SCIPY_BUILD_DEP"
+          BUILD_DEPENDS="$CYTHON_BUILD_DEP"
```

Comment:
if multibuild is calling `pip wheel`, then do we need to specify a build depend here?

Reply:
Particularly that if it uses PEP517, this would have no effect. Maybe we should change pin the cython version in pyproject.toml to     `"Cython>=0.28.5,<3.0.0"` ?

Reply:
> if multibuild is calling pip wheel, then do we need to specify a build depend here?

Wheel is already specified in pyproject.toml.

Reply:
> Maybe we should change pin the cython version in pyproject.toml to "Cython>=0.28.5,<3.0.0" ?

This was my concern as well.

`multibuild` does something equivalent to this:

```bash
pip install $BUILD_DEPENDS
pip wheel --no-deps .
```